package.json

```diff
@@ -58,7 +58,7 @@
   },
   "license": "LicenseRef-LICENSE",
   "lint-staged": {
-    "*.{js,jsx}": [
+    "*.{js}": [
```

Comment:
I will fix the JSX styling issue.

Reply:
@pwnsdx you can use the `--no-verify` if that is causing some errors..